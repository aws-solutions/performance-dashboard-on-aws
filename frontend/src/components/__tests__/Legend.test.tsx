/*
 *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 *  SPDX-License-Identifier: Apache-2.0
 */

import { render } from "@testing-library/react";
import RenderLegendText from "../Legend";

test("renders the legend", async () => {
    const wrapper = render(RenderLegendText("Legend", { color: "red" }));
    expect(wrapper.container).toMatchSnapshot();
});
