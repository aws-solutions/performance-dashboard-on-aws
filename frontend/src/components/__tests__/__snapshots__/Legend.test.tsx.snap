// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the legend 1`] = `
<div>
  <span
    class="recharts-legend-item-text"
  >
    <button
      aria-label="Hide/unhide data for Legend"
      style="background-color: transparent; color: red; border-width: 0px;"
    >
      Legend
    </button>
  </span>
</div>
`;
